### PR TITLE
patch-callbacks_test

### PR DESCRIPTION
**Pull request by autoih**
The PR is minor to modify: locate a statement to exactly place with exception raised, remove `del` variable, and add message.

### CODE REVIEW

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Review by rchao — COMMENTED** [review]
Thanks for the PR! Can your provide more information explaining why this change? It is not apparent why the statements need to reordered.

**Comment by autoih** [comment]
It may be good to locate the exact statement where the exception is raiased; to limit the number of statements in exception raising test blocks to 1.

**Comment by gbaned** [comment]
@rchao Can you please take a look on this PR? Thanks!

File: tensorflow/python/keras/activations_test.py [hunk 1]
```diff
@@ -149,8 +149,8 @@ def ref_hard_sigmoid(x):
     x = keras.backend.placeholder(ndim=2)
     f = keras.backend.function([x], [keras.activations.hard_sigmoid(x)])
     test_values = np.random.random((2, 5))
-    result = f([test_values])[0]
     expected = hard_sigmoid(test_values)
+    result = f([test_values])[0]
```
Comment (rchao) [inline tensorflow/python/keras/activations_test.py]: Can you provide the rationale for this change?

File: tensorflow/python/keras/callbacks_test.py [hunk 2]
```diff
@@ -725,12 +725,10 @@ def test_EarlyStopping_reuse(self):
           optimizer='sgd', loss='binary_crossentropy', metrics=['accuracy'])
       weights = model.get_weights()
 
-      stopper = keras.callbacks.EarlyStopping(monitor='acc', patience=patience)
```
Comment (rchao) [inline tensorflow/python/keras/callbacks_test.py]: Why removing this block of assertion?

**Review by rchao — COMMENTED** [review]
Thanks for the PR!

**Comment by gbaned** [comment]
@autoih Could you please check reviewer comments and keep us posted. Thanks!

File: tensorflow/python/keras/callbacks_test.py (hunk 2 above)
Comment (autoih) [reply tensorflow/python/keras/callbacks_test.py]: Thanks @rchao, it may be duplicate, isn't it?

**Comment by gbaned** [comment]
@autoih Can you please resolve conflicts? Thanks!

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**